puppet/data/common.yaml

```diff
@@ -241,3 +242,14 @@ sudo::wheel_config: password
 redmine::https: true
 
 apache::default_vhost: false
+
+rsync_usernames:
```

Comment:
I should have added a comment, but I started this file with custom definitions first. Then around like 12 I started to map things. With the introduction of `stable_release` I think that already broke (and I should have noticed it there).

My question: how should we manage this? Do we want to keep things that we regularly change (i.e., real data) at the top while keeping more "internals" near the bottom or keep them close to each other since they're closely connected

Reply:
Split it into two files? That would be easier for me to understand. And I imagine be more obvious to newcomers.

Reply:
Two files sounds good to me!

Reply:
I am going to treat this as a follow up and merge this.

Reply:
Implemented this idea [here](https://github.com/theforeman/foreman-infra/pull/2130)